lib/features/seller/authentication/model/seller_register_request_model.dart

```diff
@@ -0,0 +1,72 @@
+class SellerRegisterRequestModel {
+  SellerRegisterRequestModel({
+    required this.firstName,
+    required this.lastName,
+    this.displayName,
+    required this.email,
+    required this.phoneNumber,
+    required this.password,
+    required this.confirmPassword,
+    this.idToken
+  });
+
+  final String? firstName;
+  final String? lastName;
+  final String? displayName;
+  final String? email;
+  final String? phoneNumber;
+  final String? password;
+  final String? confirmPassword;
+  final String? idToken;
+
+  SellerRegisterRequestModel copyWith({
```

Comment:
In the Register response additional details also returned.
"otp": null,
        "isOtpVerified": false,
        "gstNumber": null,
        "gstOtpVerified": false,
        "storeName": null,
        "pickupAddress": null,
        "pickupLat": null,
        "pickupLng": null,
        "pickupStreet": null,
        "pickuDoorNo": null,
        "pickupCity": null,
        "pickupState": null,
        "pickupPincode": null,
        "pickupCountry": null,
        "shippingMethod": null,
        "shippingCharge": null,

Reply:
<img width="1439" alt="Screenshot 2024-12-05 at 9 44 18 AM" src="https://github.com/user-attachments/assets/29a863e3-c8c7-4323-9cf2-f301d41a81e3">


Added all the response params in the register response model.